src/common/adapters/AxelarAdapter.sol

```diff
@@ -41,22 +41,31 @@ contract AxelarAdapter is Auth, IAxelarAdapter {
     //----------------------------------------------------------------------------------------------
 
     /// @inheritdoc IAxelarAdapter
-    function file(bytes32 what, string calldata axelarId, uint16 centrifugeId, string calldata source) external auth {
+    function file(bytes32 what, string calldata axelarId, uint16 centrifugeId, string calldata source) public auth {
         if (what == "sources") sources[axelarId] = AxelarSource(centrifugeId, keccak256(bytes(source)));
         else revert FileUnrecognizedParam();
         emit File(what, axelarId, centrifugeId, source);
     }
 
     /// @inheritdoc IAxelarAdapter
     function file(bytes32 what, uint16 centrifugeId, string calldata axelarId, string calldata destination)
-        external
+        public
         auth
     {
         if (what == "destinations") destinations[centrifugeId] = AxelarDestination(axelarId, destination);
         else revert FileUnrecognizedParam();
         emit File(what, centrifugeId, axelarId, destination);
     }
 
+    /// @inheritdoc IAxelarAdapter
+    function setRemoteAdapter(uint16 centrifugeId, string calldata axelarId, string calldata adapter) external auth {
```

Comment:
I would prefer this to be a method in `Guardian`, just to keep the scope of the adapters smaller. Ultimately it's the Guardian's limitation to only be able to initialize new adapters.